repos/system_upgrade/common/libraries/distro.py

```diff
@@ -16,3 +19,206 @@ def get_distribution_data(distribution):
         raise StopActorExecutionError(
             'Cannot find distribution signature configuration.',
             details={'Problem': 'Distribution {} was not found in {}.'.format(distribution, distributions_path)})
+
+
+_DISTRO_REPOFILES_MAP = {
+    'rhel': {
+        '8': {'/etc/yum.repos.d/redhat.repo': 'all'},
+        '9': {'/etc/yum.repos.d/redhat.repo': 'all'},
+        '10': {'/etc/yum.repos.d/redhat.repo': 'all'},
+    },
+    'centos': {
+        '8': {
+            # TODO is this true on all archs?
+            'CentOS-Linux-AppStream.repo': 'all',
```

Comment:
The `'all'` could be changed for empty list or perhaps `['noarch']` or something else.